build-system/tasks/lint.js

```diff
@@ -123,11 +123,7 @@ function summarizeResults(results, fixedFiles) {
  * @return {Promise<void>}
  */
 async function lint() {
-  const filesToCheck = getFilesToCheck(
-    lintGlobs,
-    {gitignore: true},
-    '.eslintignore'
-  );
+  const filesToCheck = getFilesToCheck(lintGlobs, {}, '.eslintignore');
```

Comment:
Why the code change here?

Reply:
`gitignore` is a `globby` add-on option, and not a native `fast-glob` option. In this case, I've already made sure that `.eslintignore` contains all the correct entries, and this code change is doing exactly what we want (and no more).